src/styles/button.css

```diff
@@ -13,6 +13,7 @@
   text-transform: uppercase;
   white-space: nowrap;
   border: 0;
+  cursor: pointer;
```

Comment:
FYI @HugoGiraudel - updated CSS to show a pointer when hovering over the buttons (previously links). Let us know if you have any issues or concerns.

Reply:
Alright. :)